docs/topics/auth/default.txt

```diff
@@ -945,7 +945,7 @@ All authentication views
 This is a list with all the views ``django.contrib.auth`` provides. For
 implementation details see :ref:`using-the-views`.
 
-.. function:: login(request, template_name=`registration/login.html`, redirect_field_name='next', authentication_form=AuthenticationForm, extra_context=None, redirect_authenticated_user=False)
+.. function:: login(request, next_page=None, template_name=`registration/login.html`, redirect_field_name='next', authentication_form=AuthenticationForm, extra_context=None, redirect_authenticated_user=False)
```

Comment:
The function-based view isn't modified (and shouldn't be).

Reply:
It's no longer the case after 2dacc2ccd9593678f19b61ecfe1bc6ee826c2a88, so yes, please remove it.